docs/concepts/runtime.md

```diff
@@ -28,12 +28,12 @@ under the License.
 
 ## Tasks and Operator Chains
 
-For distributed execution, Flink *chains* operator subtasks together into *tasks*. Each task is executed by one thread.
+For distributed execution, Flink *chains* operator subtasks together into *tasks*, with one thread executing each task.
 Chaining operators together into tasks is a useful optimization: it reduces the overhead of thread-to-thread
-handover and buffering, and increases overall throughput while decreasing latency.
-The chaining behavior can be configured; see the [chaining docs](../dev/datastream_api.html#task-chaining-and-resource-groups) for details.
+handover and buffering and increases overall throughput while decreasing latency.
+You can configure the chaining behavior, read the [chaining docs](../dev/datastream_api.html#task-chaining-and-resource-groups) for details.
 
-The sample dataflow in the figure below is executed with five subtasks, and hence with five parallel threads.
+Five subtasks execute the sample data flow in the figure below with five parallel threads.
```

Comment:
The community as well as other projects refer to "dataflow" extensively (also below).